src/main/java/spark/Request.java

```diff
@@ -240,7 +240,7 @@ public String body() {
     private void readBody() {
 		try {
 			bodyAsBytes = IOUtils.toByteArray(servletRequest.getInputStream());
-			body = IOUtils.toString(new ByteArrayInputStream(bodyAsBytes));
+			body = new String(bodyAsBytes);
```

Comment:
both of these bother me, since we're relying on the default encoding for the system.